### PR TITLE
Issue/10649 material reader post

### LINKED ISSUE #10649: Implement Material theme on Reader and related screens
We should switch widgets used at Reader and descendent screens to their material counterparts, remove color references where possible, and switch to using material `textAppearance` attributes.

### PR DESCRIPTION
**Pull request by khaykov**
Fixes #10649 

This PR is based on #11164, which needs to be merged first.

This PR moves reader post details and nested screens to the material theme and adds a dark mode support.

comparison_reader_post

To test:
- Navigate to post details, post comments, and list of users who liked it. Try to like/bookmark posts and make sure everything looks and works ok in both light and dark themes.
The theme can be changed from app settings.

PR submission checklist:

- [ ] I have considered adding unit tests where possible.
- [ ] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by khaykov** [comment]
@mattmiklic I updated the color to use high type alpha 👍 
Btw, the author's name and site name are actionable - tapping on it opens their blog. Do you still want to use a high type onSurface for them?

comparison_post_detail_text_color

**Comment by mattmiklic** [comment]
I think that's ok... tapping the user's name isn't the primary action, so I don't think we want to make it compete with the Follow action visually. That latest screenshot looks good to me.

File: WordPress/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -17,7 +17,9 @@
     <dimen name="fab_margin_tablet">24dp</dimen>
     <dimen name="design_fab_size">56dp</dimen>
 
-    <item format="float" name="primary_button_disabled_alpha" type="dimen">0.12</item>
+    <item name="primary_button_disabled_alpha" format="float" type="dimen">@dimen/disabled_alpha
+    </item>
```
Comment (develric) [inline WordPress/src/main/res/values/dimens.xml]: Maybe I remember this is the way AS auto format this sort of things and this is less than a np, it's more a proof that I went into the code 😬! I would align on a single line, but really super minor so feel free to just ignore this!

Comment (khaykov) [reply WordPress/src/main/res/values/dimens.xml]: I switched formatting to multiline, otherwise AS will reformat it again in te future :)

**Comment by jkmassel** [comment]
We're freezing `14.2` today, so this PR is being bumped to `14.3`. If you'd like it to ship with `14.2`, please merge it into `release/14.2` and ping me – I'll be happy to cut a new beta release.

**Pull request merged**